src/Umbraco.Core/Constants-Conventions.cs

```diff
@@ -118,6 +118,26 @@ public static class MediaTypes
                 /// </summary>
                 public const string Image = "Image";
 
+                /// <summary>
+                /// MediaType alias for an video-.
+                /// </summary>
+                public const string Video = "Video";
```

Comment:
One thing, I just thought reading through this draft thing (exciting, I like the icon for sound!) would people already have custom MediaTypes in their project with these Alias... ? just thinking about upgrades - I know I always use 'Video' for alias of a custom MediaType that points to an external video resource... anyway as the thought occurred, I thought I'd say it out loud, but sorry, if I've misconstrued!

Reply:
Perfect insight, thanks for sharing this. The new Media Types only gets created for new installs, so shouldn't be a concern. Existing installations will have to setup the types on their own.
If I´m not mistaken the configuration of file extensions accepted by a Upload-field is already part of the pre-values in the latest release.